.F.
======================================================================
FAIL: test_parser (__main__.TestPythonDebug)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "d:\g\fhopecc\test_python_debug.py", line 9, in test_parser
    self.assertEqual('d', 'www')
AssertionError: 'd' != 'www'
- d
+ www


----------------------------------------------------------------------
Ran 3 tests in 0.076s

FAILED (failures=1)
